feeder/cmd/pixel_bt_feeder/standalone.conf/feeder.yaml

```diff
@@ -0,0 +1,10 @@
+Log:
```

Comment:
sideways thought: if we update the pixel feeder to technically support multiple logs here (and rename the witness public key related properties), we could merge this and the pixelwitness.yaml files together.

Not sure if this is end-of-day-tired-madness talking here...

Reply:
This could be interesting. It doesn't make this any easier to configure/deploy, so I'll defer it from this PR but let's discuss as its own thread.